### PR TITLE
prepare for 1.0.6

### PR DESCRIPTION
**Pull request by jschlyter**
It may be time to release 1.0.6 with clarifications only?

Please check html/docx generation targets! `make all` should generate both html and docx (requires grip and pandoc).

### CODE REVIEW

File: hcert_spec.md [hunk 1]
```diff
@@ -108,9 +109,9 @@ Verifiers MAY apply additional policies with the purpose of restricting the vali
 
 #### 3.3.7 Health Certificate Claim
 
-The Health Certificate (`hcert`) claim is a JSON ([RFC 7159](https://tools.ietf.org/html/rfc7159)) object containing the health status information, which has been encoded and serialised using CBOR as defined in ([RFC 7049](https://tools.ietf.org/html/rfc7049)). Several different types of health certificate MAY exist under the same claim, of which the European DGC is one.
+The Health Certificate (`hcert`) claim is a JSON ([RFC 7159](https://tools.ietf.org/html/rfc7159)) object containing the health status information. Several different types of health certificate MAY exist under the same claim, of which the European DGC is one.
 
-Note here that the JSON is purely for schema purposes. The wire format is CBOR. Application developers may not actually ever de-, or encode to and from the JSON format, but use the in-memory structure.
+Note here that the JSON is purely for schema purposes. The wire format is CBOR as defined in ([RFC 7049](https://tools.ietf.org/html/rfc7049)). Application developers may not actually ever de-, or encode to and from the JSON format, but use the in-memory structure.
```
Comment (martin-lindstrom) [inline hcert_spec.md:114]: "de-, or encode" -> decode or encode

**Review by martin-lindstrom — APPROVED** [review]

File: hcert_spec.md (hunk 1 above)
Comment (jschlyter) [reply hcert_spec.md]: Fixed in e6641a4b2ea8caf074b181def86b39bff3bb75b7

**Review by martin-lindstrom — APPROVED** [review]

File: hcert_spec.md [hunk 2]
```diff
@@ -84,7 +85,7 @@ This corresponds to the COSE algorithm parameter: `PS256`.
...
+The Key Identifier (`kid`) claim is used by Verifiers for selecting the correct public key from a list of keys pertaining to the Issuer (`iss`) Claim. Several keys may be used in parallel by an Issuer for administrative reasons and when performing key rollovers. The Key Identifier is not a security-critical field. For this reason, it MAY also be placed in an unprotected header if required. Verifiers MUST accept both options.  If both options are present, the Key Identifier in the protected header MUST be used.
 
 Due to the shortening of the identifier (for space-preserving reasons) there is a slim but non-finite chance that the overall list of DSCs accepted by a validator may contain DSCs with duplicate `kid`s. For this reason a verifier MUST check all DSCs with that `kid`.
```
Comment (goebelUB) [inline hcert_spec.md:90]: Shouldn't this read "non-infinite chance"?

Comment (jschlyter) [reply hcert_spec.md]: Fixed in 508941f99c55af8b93761d25646e1852acd729e8

**Review by martin-lindstrom — DISMISSED** [review]

File: hcert_spec.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Electronic Health Certificate Specification
 
-Version 1.0.5, 2021-04-18
+Version 1.0.6, 2021-04-28
```
Comment (fredriklj) [inline hcert_spec.md:3]: Bump date?

**Review by fredriklj — DISMISSED** [review]

**Review by martin-lindstrom — APPROVED** [review]

**Review by fredriklj — APPROVED** [review]

**Pull request merged**